.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
can we have a more semver like pattern here so it doesn't have false positive if tag was verbose-fix